app/views/layouts/application.html.slim

```diff
@@ -2,5 +2,6 @@ doctype html
 html
   == render 'application/head'
   body
-  .container
+    = render 'shared/header'
+    = render 'shared/footer'
     == yield
```

Comment:
Prefer to render `footer` in the end of the layout (after `yield`ing of a templates)